calyx-py/calyx/builder.py

```diff
@@ -789,10 +793,39 @@ def static_if(
     else_body=None,
 ) -> ast.If:
     """Build an `if` control statement."""
-    else_body = ast.Empty() if else_body is None else else_body
+    else_body = else_body or ast.Empty()
```

Comment:
Looks like the comment on this is also wrong. Maybe they got flipped around in the past?

Reply:
I was wondering about this. Now fixed!